src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -34,17 +37,35 @@ class Creator(Mediator):
     allowed_origins = Param(Iterable, default=lambda self: [])
     request = Param("rest_framework.request.Request", required=False)
     user = Param("sentry.models.User")
+    is_internal = Param(bool, default=False)
```

Comment:
Nit: [is `False` a reasonable default](https://softwareengineering.stackexchange.com/a/63929)? My sense is that there isn't a reasonable default so each of the app creators should specify if it's internal or not.